core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -57,8 +61,8 @@ var ExplorationEditorFeedbackTab = function() {
           return suggestionDescription.indexOf(subject) !== -1;
         });
       });
-      expect(matchingSuggestionRows[0].isDisplayed()).toBe(true);
-      matchingSuggestionRows[0].click();
+      expect(matchingSuggestionRows[1].isDisplayed()).toBe(true);
```

Comment:
Hey @DubeySandeep 
Since we added the reject suggestion check before the accept check, the rejected suggestion becomes the first, because of which we had to increase the index here by 1.

Reply:
I see, thanks for the explanation! :)